eth2near/eth2near-block-relay-rs/src/execution_block_proof.rs

```diff
@@ -129,6 +129,8 @@ mod tests {
 
     const TEST_BEACON_BLOCK_ID: u32 = 741888;
     const BEACON_ENDPOINT: &str = "https://lodestar-kiln.chainsafe.io";
+    const TIMEOUT: u64 = 30;
```

Comment:
The same here `TIMEOUT_SECONDS`

Reply:
fixed: https://github.com/aurora-is-near/rainbow-bridge/pull/800/commits/4aefac2a955839abc6233e6c63efe671be60feef